doc/Admin Documentation.md

```diff
@@ -4,6 +4,23 @@
 
 Here you can find the configuration instructions for the currently supported gateways.
 
+## CustomSMS
+URL: any endpoint
+Stability: Experimental
```

Comment:
I think that will be good implement examples of supported API payload with GET and POST.

For example:

Provider with GET:
Need accept URL on this format: `<base_url_provider>?<mobile_number_parameter>=<number> ....`

Provider with POST:
...

Mandatory parameters:
| base_url_provider | the base url of provider |
|---|---|
| mobile_number_parameter | phone number parameter, get from provider, by example: phone, number, num, etc |